src/ui.py

```
@@ -1,22 +1,22 @@
 import time
-import sys
-from colorama import Fore, Style, init
+from rich.console import Console
+from rich.panel import Panel
+from rich.table import Table
+from rich.text import Text
```

Ruff (0.8.2)

`rich.text.Text` imported but unused

Remove unused import: `rich.text.Text`

(F401)